mappings/net/minecraft/client/render/block/entity/BlockEntityRenderDispatcher.mapping

```diff
@@ -14,23 +14,33 @@ CLASS net/minecraft/class_824 net/minecraft/client/render/block/entity/BlockEnti
 		ARG 4 light
 		ARG 5 overlay
 	METHOD method_23078 register (Lnet/minecraft/class_2591;Lnet/minecraft/class_827;)V
+		ARG 1 type
+		ARG 2 renderer
 	METHOD method_23079 render (Lnet/minecraft/class_827;Lnet/minecraft/class_2586;FLnet/minecraft/class_4587;Lnet/minecraft/class_4597;)V
 		ARG 0 renderer
 		ARG 1 blockEntity
+		ARG 2 tickDelta
+		ARG 3 matrices
+		ARG 4 vertexConsumerProvider
 	METHOD method_3549 configure (Lnet/minecraft/class_1937;Lnet/minecraft/class_1060;Lnet/minecraft/class_327;Lnet/minecraft/class_4184;Lnet/minecraft/class_239;)V
 		ARG 1 world
 		ARG 2 textureManager
 		ARG 3 textRenderer
 		ARG 4 camera
 		ARG 5 crosshairTarget
 	METHOD method_3550 get (Lnet/minecraft/class_2586;)Lnet/minecraft/class_827;
+		ARG 1 blockEntity
 	METHOD method_3551 setWorld (Lnet/minecraft/class_1937;)V
+		ARG 1 world
 	METHOD method_3552 renderEntity (Lnet/minecraft/class_2586;Lnet/minecraft/class_4587;)V
 		ARG 1 entity
 		ARG 2 matrix
 	METHOD method_3554 runReported (Lnet/minecraft/class_2586;Ljava/lang/Runnable;)V
+		ARG 0 blockEntity
+		ARG 1 task
 	METHOD method_3555 render (Lnet/minecraft/class_2586;FLnet/minecraft/class_4587;Lnet/minecraft/class_4597;)V
 		ARG 1 blockEntity
 		ARG 2 tickDelta
-		ARG 3 matrix
+		ARG 3 matrices
+		ARG 4 vertexConsumerProvider
```

Comment:
This can be `vertexConsumers`.

Reply:
Aren't most of the param usages called `vertexConsumerProvider`?

Reply:
No, the `VertexConsumerProvider` was renamed to that in my last PR, and all parameters I named are `vertexConsumers`.